scene/gui/color_picker.cpp

```diff
@@ -1735,11 +1735,11 @@ ColorPicker::ColorPicker() {
 	btn_pick = memnew(Button);
 	sample_hbc->add_child(btn_pick);
 	if (DisplayServer::get_singleton()->has_feature(DisplayServer::FEATURE_SCREEN_CAPTURE)) {
-		btn_pick->set_tooltip_text(RTR("Pick a color from the screen."));
+		btn_pick->set_tooltip_text(ATR("Pick a color from the screen."));
```

Comment:
It won't work in the constructor because unchecking "Auto Translate" is done after the node is created.